editor/src/components/canvas/canvas-strategies/draw-to-insert-strategy.spec.browser2.tsx

```diff
@@ -60,6 +59,31 @@ async function fireDragEvent(
   })
 }
 
+async function fireClickEvent(canvasControlsLayer: HTMLElement, point: WindowPoint) {
+  await act(async () => {
+    fireEvent(
+      canvasControlsLayer,
+      new MouseEvent('mousedown', {
+        bubbles: true,
+        cancelable: true,
+        clientX: point.x,
+        clientY: point.y,
+        buttons: 1,
+      }),
+    )
+
+    fireEvent(
```

Comment:
I wonder if it'd be worth dispatching a click as well (because in real life a click event would follow). to make the test less synthetic. but I know it can never be 100% simulation because you are dispatching the events only on the canvasControlsLayer

Reply:
I've mentally wrestled with this one too. We really should be dispatching all of the events that would fire (we should probably just write helper functions since we do this so much anyway), but it will still be a very artificial situation. Though at the least we do know that the events will bubble, so it will still be closer to real life anyway.

Reply:
Ok, I added dispatching click event too in d86dfd425b6faad711e0fb82164b33eb0cfc71d0